src/main/webapp/video.js

```diff
@@ -45,3 +43,30 @@ function onYouTubeIframeAPIReady() {
 function onPlayerReady(event) {
   event.target.playVideo();
 }
+
+/** Starts timer if {@code currentState} is playing, stops otherwise. */
+function startStopTimerOnStateChange(currentState) {
+  if (currentState.data == window.YT.PlayerState.PLAYING) {
+    startTimer();
+    return;
+  }
+  stopTimer();
+}
+
+/** Starts timer which calls {@code sync} with the current video time.  */
+function startTimer() {
+  window.timer = window.setInterval(() => {
```

Comment:
can we call this `videoSyncTimer` or something a little less general

Reply:
also consider renaming the start/stop functions.  unfortunately they are added to the global/window scope, and i worry there could be name collisions with other timers in the future